avm/ptn/sa/modernize-your-code/tests/e2e/enterprise-grade/main.test.bicep

```
@@ -0,0 +1,59 @@
+targetScope = 'subscription'
+
+metadata name = 'Default configuration with WAF aligned parameter values'
+metadata description = 'This instance deploys the [Modernize Your Code Solution Accelerator](https://github.com/microsoft/Modernize-Your-Code-Solution-Accelerator) using parameters that deploy the WAF aligned configuration.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-waf-${namePrefix}-sa.moderncode-${serviceShort}-rg'
```

waf can be removed both to avoid mentioning the wording and because serviceShort already differentiates the unique name
```suggestion
param resourceGroupName string = 'dep-${namePrefix}-sa.moderncode-${serviceShort}-rg'
```